### PR TITLE
Add network security related domains

### PR DESCRIPTION
**Pull request by sinlee1**

### CODE REVIEW

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
1. use `category-network-security` as filename.
2. add `@cn` attr or include the whole file in the `geolocate-cn` category. (In that case, the filename should tell it ONLY contains Chinese sites)

File: data/network_security [hunk 1]
```diff
@@ -0,0 +1,4 @@
+fofa.so
+t00ls.cc
+t00ls.net
+tools.team
```
Comment (IceCodeNew) [inline data/network_security:4]: BTW, can you provide the accessible sites' URLs under this domain?

Comment (sinlee1) [reply data/network_security:4]: 1. www.fofa.so
2. www.t00ls.cc
3. www.t00ls.net

Both are official websites

**Review by sinlee1 — COMMENTED** [review]
Changed

File: data/network_security (hunk 1 above)
Comment (IceCodeNew) [reply data/network_security]: > * [www.fofa.so](http://www.fofa.so)
> * [www.t00ls.cc](http://www.t00ls.cc)
> * [www.t00ls.net](http://www.t00ls.net)
> 
> Both are official websites

I mean `*.tools.team`

Comment (sinlee1) [reply data/network_security]: - About fofa.so

`classic.fofa.so`

`faweb.fofa.so`

- About `t00ls.cc` `t00ls.net` `tools.team` 

These three domains are all the same official site, just with different domain names

These 3 domains do not have subdomains, only www domains

Comment (IceCodeNew) [reply data/network_security]: image

Comment (sinlee1) [reply data/network_security]: - Situation description

No need to add www, you can directly T00ls.team for access.

The domain name will be redirected to t00ls.cc after 302 hops

Also attached is the official announcement

image

**Review by IceCodeNew — APPROVED** [review]

**Comment by IceCodeNew** [comment]
Thanks for your contribution!

**Pull request merged**